internal/ec2provider/compute.go

```diff
@@ -34,22 +34,21 @@ type ComputeManager interface {
 }
 
 type computeManager struct {
-	region    string
-	ec2Client *awsutils.Client
+	clientTimeoutSeconds int
```

Comment:
I think the `clientTimeoutSeconds` should be moved to the new config and be called something like "awsClientTimeoutSeconds" (e.g. the S3 client uses it and S3 is not compute).

Instead of having to pass it around, each client can just obtain it directly from config (this was the motivation behind merging that PR first).